### DISCUSSION TITLE
Wrong language

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#399] [Q&A (Community Support)]
Hello, I just wanted to inform you that this is the wrong language. This should be HR - Croatian - Hrvatski
![Akeeba-Panopticon](https://github.com/akeeba/panopticon/assets/17553018/f0edb57a-5382-492a-9db5-01a846ca51e7)

**Comment by nikosdion** [comment]
Well, the translation of the language name originally was "English", but in that other language, so I had to guess. I'll change that.
